src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form form-big"
+          data-qa="big"
+          action="/">
```

Comment:
Check the code-style rules: [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/933e84b4-f143-4a2e-b2be-6d904b271790)